src/Tools/dotnet-monitor/ConsoleOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Options
+#else
+namespace Microsoft.Diagnostics.Tools.Monitor
+#endif
+{
+    internal sealed class ConsoleOptions
```

Comment:
Should we really be redefining these types? Wouldn't it be preferable to use existing types and add them into the schema generator unit test?

Reply:
One issue that I encountered while reusing existing types for a different feature was that the schema generation will generate base schemas for the base types but I couldn't get VS Code IntelliSense to work with it; it's possible I did something wrong when trying it.

Also, we'd have to manually insert descriptions and default values for the entire structure in the schema generator project, which isn't ideal.

A minor benefit to the way it is now is that it allows tests to change logging levels through a strongly-type mechanism. They currently do it via environment variables with well-known names, which is fine to do because how logging is done isn't really going to change. A change in the category name or level would impact the tests regardless of how they are described.

Reply:
Then perhaps we can only compile them into the test code? We don't need these in the actual dotnet-monitor code.